src/components/Spinner/index.js

```diff
@@ -6,23 +6,42 @@ import StyledArcSpinner from './styled/arcSpinner';
 import StyledSpinnerContainer from './styled/spinnerContainer';
 import StyledChildContainer from './styled/childContainer';
 
+const sizeMap = {
```

Comment:
since you are using the same size map in StyledArcSpinner i think you could put it in a separate file and import it from both places